cmd/agent/app/options/options.go

```diff
@@ -82,6 +88,9 @@ func NewOptions() *Options {
 			LeaderElect:       true,
 			ResourceLock:      resourcelock.LeasesResourceLock,
 			ResourceNamespace: util.NamespaceKarmadaSystem,
+			LeaseDuration:     defaultElectionLeaseDuration,
+			RenewDeadline:     defaultElectionRenewDeadline,
+			RetryPeriod:       defaultElectionRetryPeriod,
```

Comment:
I think we don't need it since it will be set when initializing the flag.

Reply:
got it!